Framework/DataHandling/src/LoadMuonNexus.cpp

```diff
@@ -101,6 +101,14 @@ void LoadMuonNexus::init() {
                                                                  PropertyMode::Optional),
                   "Table or a group of tables with information about the "
                   "detector grouping stored in the file (if any). Version 1 only.");
+
+  declareProperty(std::make_unique<ArrayProperty<std::string>>("PeriodsInformation"),
```

Comment:
Quick Q: Is this an output property? If so we should mark it as one so that the Python signature is generated correctly.

Reply:
Could I ask if the correct way to do this is like so:
```
  declareProperty("PeriodsInformation", std::make_unique<ArrayProperty<std::string>>("PeriodsInformation"),
                  "An array of strings which hold period information. This array is in the "
                  "format ['NumberOfSequences', 'Labels', 'Type', 'Frames', "
                  "'TotalFrames', 'Counts', 'Tag,]. Empty "
                  "strings "
                  "show no value "
                  "could be read from file.", Direction::Output);
```

Reply:
Yep that's it 👍🏻